test/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepositoryTest.java

```diff
@@ -0,0 +1,313 @@
+                .setId(uuid)
+                .setCluster("new-cluster")
+                .build());
+        models.ebean.Alert ebeanAlert1 = Ebean.find(models.ebean.Alert.class)
+                .where()
+                .eq("uuid", uuid)
+                .findUnique();
+        models.ebean.Alert ebeanAlert2 = Ebean.find(models.ebean.Alert.class)
+                .where()
+                .eq("uuid", uuid)
+                .findUnique();
+        try (Transaction transaction = Ebean.beginTransaction()) {
+            ebeanAlert1.setCluster("new-cluster1");
+            ebeanAlert2.setCluster("new-cluster2");
+            ebeanAlert2.save();
+            ebeanAlert1.save();
+            transaction.commit();
+        } catch (final IOException e) {
+            //Do Nothing
+        }
+    }    private final DatabaseAlertRepository.AlertQueryGenerator queryGenerator = new DatabaseAlertRepository.GenericQueryGenerator();
+    private final DatabaseAlertRepository alertRepo = new DatabaseAlertRepository(queryGenerator);
+    private final DefaultAlert.Builder alertBuilder = new DefaultAlert.Builder()
+            .setId(UUID.randomUUID())
+            .setCluster(TEST_CLUSTER)
+            .setMetric(TEST_METRIC)
+            .setContext(TEST_CONTEXT)
+            .setService(TEST_SERVICE)
+            .setExtensions(TEST_EXTENSIONS)
+            .setName(TEST_NAME)
+            .setOperator(TEST_OPERATOR)
+            .setPeriod(Period.seconds(TEST_PERIOD_IN_SECONDS))
+            .setStatistic(TEST_STATISTIC)
+            .setValue(TEST_QUANTITY);
+
+    private static final String TEST_CLUSTER = "test-cluster";
+    private static final String TEST_METRIC = "test-metric";
+    private static final String TEST_SERVICE = "test-service";
+    private static final Context TEST_CONTEXT = Context.CLUSTER;
+    private static final Map<String, Object> TEST_EXTENSIONS = ImmutableMap.of(
+            "severity", "CRITICAL",
+            "notify", "abc@groupon.com",
+            "max_check_attempts", 3,
+            "freshness_threshold", 5);
+    private static final String TEST_NAME = "test-name";
+    private static final Operator TEST_OPERATOR = Operator.EQUAL_TO;
+    private static final int TEST_PERIOD_IN_SECONDS = 600;
+    private static final String TEST_STATISTIC = "test-statistic";
+    private static final double TEST_QUANTITY_VALUE = 12.0;
+    private static final String TEST_QUANTITY_UNIT = "test-unit";
+    private static final Quantity TEST_QUANTITY = new DefaultQuantity.Builder()
+            .setValue(TEST_QUANTITY_VALUE)
+            .setUnit(TEST_QUANTITY_UNIT)
+            .build();
+    private static final NagiosExtension TEST_NAGIOS_EXTENSIONS = new NagiosExtension();
+    private static final int BASE_PORT = 50000;
+    private static final Random RANDOM = new Random();
+
+    static {
+        TEST_NAGIOS_EXTENSIONS.setSeverity("CRITICAL");
+        TEST_NAGIOS_EXTENSIONS.setNotify("abc@groupon.com");
```

Comment:
You shouldn't use a groupon address here. Please use example.com as the domain (for example).